plotIt_ZA/groups.yml

```diff
@@ -20,135 +20,25 @@
   legend: 'Data'
 'Higgs':
   legend: 'SM Higgs'
```

Comment:
actually it's only TTH and VH no ?

Reply:
Yes it is... is there anything to change?

Reply:
a more explicit legend ! `ttH, VH` should do, we don't consider ggH, VBF, bbH